Netro Sprinklers.indigoPlugin/Contents/Server Plugin/plugin.py

```
@@ -719,86 +748,99 @@ def _update_from_netro(self):
-                dev.updateStatesOnServer(update_list)
+                # --- Schedules (only when info also fires) ---
+                schedule_dict = None
+                active_schedule_name = None
+                if now >= self._next_schedules_update:
+                    try:
+                        schedule_dict = self.api_client.get_schedules(key, api_version=api_version)
+                        schedule_states, active_schedule_name = self.sprinkler_handler.process_schedules(
+                            schedule_dict, api_version=api_version
+                        )
+                        update_list.extend(schedule_states)
+                    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)